README.md

```diff
@@ -1,7 +1,55 @@
+# "Azure App Configuration Sync

```

Comment:
Extra quote in the beginning.

Reply:
Sent separate PR to remove the extra quote